### DISCUSSION TITLE
"Could not find libuv_LIBRARY" error on Windows 10

### DISCUSSION THREAD

**Discussion by nhansendev** [libuv/libuv#3421] [Q&A]
I am attempting to install the horovod package via pip on Windows 10 (Python 3.7), which has a dependency on **libuv**.  During the install process I receive the following error:
```
CMake Error at third_party/gloo/cmake/Dependencies.cmake:46 (find_library):
      Could not find libuv_LIBRARY using the following names: uv, libuv
    Call Stack (most recent call first):
      third_party/gloo/CMakeLists.txt:109 (include)
```

I have installed libuv per the instructions for Windows and successfully ran the two tests, so I'm assuming I just need to provide a path to this library for cmake. Can anyone help me set this up, please?

**Comment by cjihrig** [comment]
Is this with version 1.43.0 of libuv? If so, there is a fix in https://github.com/libuv/libuv/pull/3415.

**Reply by nhansendev** [reply]
I was using the latest version from a day or so ago. Thank you for pointing me to the fix. I will check if it fixes the issue for me.
